### PR TITLE
Create textual-practice.csl

### PR DESCRIPTION
**Pull request by darwinev0lved**

### CODE REVIEW

File: textual-practices.csl [hunk 1]
```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB">
+  <info>
+    <title>Textual Practices</title>
+    <title-short>TextPrac</title-short>
+    <id>http://www.zotero.org/styles/textual-practices</id>
+    <link href="http://www.zotero.org/styles/textual-practices" rel="self"/>
+    <author>
```
Comment (POBrien333) [inline textual-practices.csl]: Could you add here also the style you used as a template.
Could you also add here a link to the style guidelines using rel="documentation"?

File: textual-practices.csl [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+      <name>Jon Rubin</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <issn>1121-2179</issn>
+    <eissn>1825-6562</eissn>
+    <summary>Textual Practices journal’s citation style: footnotes plus IBID citation style</summary>
+    <published>2014-01-24T00:00:00+00:00</published>
+    <updated>2018-05-22T00:59:23+00:00</updated>
```
Comment (POBrien333) [inline textual-practices.csl]: Could you changed the published date to when you made the style (something in Feb 2019) and remove the updated line?

File: textual-practices.csl [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+    <title>Textual Practices</title>
+    <title-short>TextPrac</title-short>
+    <id>http://www.zotero.org/styles/textual-practices</id>
+    <link href="http://www.zotero.org/styles/textual-practices" rel="self"/>
+    <author>
+      <name>Jon Rubin</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <issn>1121-2179</issn>
```
Comment (POBrien333) [inline textual-practices.csl]: Can you add the correct ISSNs for the journal?

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**